kolibri_exercise_perseus_plugin/assets/src/widgetSolver/index.js

```diff
@@ -1,3 +1,4 @@
+import logger from 'kolibri.lib.logging';
```

Comment:
Is your IDE very opinionated about import order? Should we add a linting rule?

Reply:
Already added haha https://github.com/learningequality/kolibri/pull/3471

Reply:
Ah, I missed that - you must love that blame!